# Project 1

__ The chart above describes arrival delays for two airlines across five destinations. Your task is to __

- (1) Create a .CSV file (or optionally, a MySQL database!) that includes all of the information below. You may change the structure of the information as you like.
- (2) Read the information from your .CSV file into pandas, and perform analysis to compare the arrival delays for the two airlines.
- (3) Your code should be in a Jupyter (IPython) notebook (in GitHub), and should include narrative descriptions of your analysis and conclusions.

### 1. Import pandas

In [3]:
import pandas as pd
import numpy as np

### 2. Reading the information from the .CSV file into pandas

In [19]:
d = pd.read_csv('Delays.csv')
d

Airlines   Status  Los Angeles  Phoenix  San Diego  San Francisco  Seattle
0   ALASKA  on time          497      221        212            503     1841
1   ALASKA  delayed           62       21         20            102      305
2  AM WEST  on time          694     4849        383            320      201
3  AM WEST  delayed          117      415         65            129       61

### 3. Perform analysis to compare the arrival delays for the two airlines

In [20]:
d['TotalFlights'] = (d['Los Angeles'] + d['Phoenix'] + d['San Diego'] + d['San Francisco'] + d['Seattle'])
d

Airlines   Status  Los Angeles  Phoenix  San Diego  San Francisco  Seattle  \
0   ALASKA  on time          497      221        212            503     1841   
1   ALASKA  delayed           62       21         20            102      305   
2  AM WEST  on time          694     4849        383            320      201   
3  AM WEST  delayed          117      415         65            129       61   

   TotalFlights  
0          3274  
1           510  
2          6447  
3           787

### 4. Calculating the percentage of the delay on each Airlines

In [22]:
# This code finds out the total number of flights and the percentage of delayed flights of Alaska Airlines
Totalflights= (d['TotalFlights'][0] + d['TotalFlights'][1])
DelayPercent= (d['TotalFlights'][1] / d['TotalFlights'][0]*100) 
print ('Total number of flights Alaska airlines had = {} '.format(Totalflights))
print ('Alaska airline delay percentage of flights = {}'.format(DelayPercent))

Total number of flights Alaska airlines had = 3784 
Alaska airline delay percentage of flights = 15.577275503970679


In [23]:
# This code finds out the total number of flights and the percentage of delayed flights of AM West Airlines
Totalflights= (d['TotalFlights'][2] + d['TotalFlights'][3])
DelayPercent= (d['TotalFlights'][3] / d['TotalFlights'][2]*100) 
print ('Total number of flights AM West airlines had = {} '.format(Totalflights))
print ('AM West airline delay percentage of flights = {}'.format(DelayPercent))

Total number of flights AM West airlines had = 7234 
AM West airline delay percentage of flights = 12.20722816814022


### 5. Analysis and Conclusion

###### Alaska Airlines:
- Total number of on time flights: 3274
- Total number of delayed flights: 510
- Total number of flights: 3784
- Percentage of delayed flights: 15.6 %

###### AM West Airlines:
- Total number of on time flights: 6447
- Total number of delayed flights: 787
- Total number of flights: 7234
- Percentage of delayed flights: 12.2 %